Sources/Logging/Logging.swift

```diff
@@ -39,20 +39,44 @@ import WASILibc
 /// logger.info("Hello World!")
 /// ```
 public struct Logger {
-    @usableFromInline
-    var handler: LogHandler
+    /// A private property to hold the boxed `LogHandler`.
+    private var _handler: Storage
 
     /// An identifier of the creator of this `Logger`.
     public let label: String
+    
+    /// A computed property to access the `LogHandler`.
+    public var handler: LogHandler {
```

Comment:
Same here for perf reasons.
```suggestion
    @inlnable
    public var handler: LogHandler {
```

Reply:
@FranzBusch As the properties are internal, I am not able to use `@inlinable`.

Reply:
If you mark them as `@usableFromInline` this will work